aws/resource_aws_waf_ipset_test.go

```diff
@@ -33,6 +34,8 @@ func TestAccAWSWafIPSet_basic(t *testing.T) {
 						"aws_waf_ipset.ipset", "ip_set_descriptors.4037960608.type", "IPV4"),
 					resource.TestCheckResourceAttr(
 						"aws_waf_ipset.ipset", "ip_set_descriptors.4037960608.value", "192.0.7.0/24"),
+					resource.TestMatchResourceAttr("aws_waf_ipset.ipset", "arn",
+						regexp.MustCompile(`^arn:[\w-]+:([a-zA-Z0-9\-])+:([a-z]{2}-(gov-)?[a-z]+-\d{1})?:(\d{12})?:(.*)$`)),
```

Comment:
This regex is pretty hard to read and has some issues:
* The service name should always be `waf`
* The region name should always exist
* The account ID should always exist
* The resource should check for `ipset/.+`

At some point we'll probably create a helper function for ARN testing like this, but as for this case, I will fix these items post-merge with the simpler: `^arn:[\w-]+:waf:[^:]+:\d{12}:ipset/.+$`.

Reply:
Actually double checking the AWS documentation here: https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arn-syntax-waf

> arn:aws:waf::123456789012:ipset/3f74bd8c-f046-4970-a1a7-41aa52e05480

Looks like region name should never be set. I'll adjust the ARN logic and regex here accordingly.

In the regional case, it will include the region, but that is a separate resource and test 👍 

> arn:aws:waf-regional:us-east-1:123456789012:ipset/3f74bd8c-f046-4970-a1a7-41aa52e05480

Reply:
Yeah, correct. Will I add `arn` attribute to `aws_wafregional_ipset`, is it useful?

Reply:
@saravanan30erd I'm sure someone will eventually be looking for it! 👍

Reply:
@bflad Its done #4816 👍